Specs/DateTime/Spanish/MergedExtractor.json

```diff
@@ -1,10 +1,11 @@
 [{
   "Input": "esto es 2 dias en p ",
+  "NotSupported": "javascript",
   "NotSupportedByDesign": "python",
   "Results": [
     {
-      "Text": "2 dias",
-      "Type": "duration"
+      "Text": "2 dias en",
+      "Type": "daterange"
     }
```

Comment:
For this test case, I can't read Spanish so I'm not quite sure. But in the implementation, a duration followed by a past or future string will be extracted. Here "2 dias" is a duration, and "en" can match with the FutureRegex in Spanish, so it's recognized as a daterange.